nesta_ds_utils/file_ops.py

```diff
@@ -30,6 +30,7 @@ def make_path_if_not_exist(path: Union[Path, str]):
 
 def extractall(zip_path: Union[Path, str], out_path: Union[Path, str]=None, delete_zip: Boolean = True):
```

Comment:
I think our intention on line length is to say keep it within the style guide unless, but break the limit with any good reason. Confusingly, the Google Python style guide specifies 80 characters per line max, while Black specifies 88, so it's not like we have a hard rule on this. Saying that, I think it gets a bit unreadable when you've got lots of params like this. More importantly, it's probably worth roughly adhering to some limit as we anticipate this repo will have lots of contributors so it could get messy. I would probably make it more like but up to you!

```suggestion
def extractall(
    zip_path: Union[Path, str],
    out_path: Union[Path, str] = None,
    delete_zip: Boolean = True,
):
```